### PR TITLE
add theme docs

### PR DESCRIPTION
**Pull request by danethurber**
theme docs

### CODE REVIEW

File: docs/src/pages/components/theme.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+route: /components/theme
+---
+
+import Theme from '@pluralsight/ps-design-system-theme'
+
+import { Types, Usage } from '../../components/component'
+
+# Theme
+
+## Usage pattern
```
Comment (jaketrent) [inline docs/src/pages/components/theme.mdx:12]:
```suggestion
## Examples
```

Not sure this is necessarily required.  It's just the heading that we've used as a standard so far.

File: docs/src/pages/components/theme.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+To theme a component or part of the page, surround it in the `Theme` component.
+If you're on React < 16, you'll need to wrap a single child, so a child `div`
+may be required.
```
Comment (jaketrent) [inline docs/src/pages/components/theme.mdx:16]:
```suggestion
```

Does this apply to anyone any more?  Maybe remove?

File: docs/src/pages/components/theme.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+```typescript noRender startExpanded
```
Comment (jaketrent) [inline docs/src/pages/components/theme.mdx:18]: Why `startExpanded`?  It's long.  What makes this one special?

**Review by jaketrent — APPROVED** [review]

File: docs/src/pages/components/theme.mdx (hunk 3 above)
Comment (danethurber) [reply docs/src/pages/components/theme.mdx:18]: there isn't a preview and it's basically the only thing page. seems like this codeblock is the most important thing going on so i didn't want to hide it

**Pull request merged**